src/sequelize.d.ts

```diff
@@ -161,7 +161,7 @@ export interface Config {
   readonly protocol: 'tcp';
   readonly native: boolean;
   readonly ssl: boolean;
-  readonly replication: boolean;
+  readonly replication?: ReplicationOptions | false;
```

Comment:
```suggestion
  readonly replication: ReplicationOptions | false;
```

`replication` is optional in `Options` but not in `Config` as Sequelize will assign a default value

Reply:
@ephys fixed